tests/api_test/api_test.cpp

```diff
@@ -51,6 +51,10 @@ CATCH_REGISTER_LISTENER(_watchdog)
 
 #define WAIT_TIME_IN_MS 5000
 
+#if !defined(MAX_TAIL_CALL_CNT)
+#define MAX_TAIL_CALL_CNT 33
+#endif
```

Comment:
I think these lines should be deleted and the define instead added (with no #if) into ebpf_structs.h

Reply:
Thanks.

In addition, the below exposed in bpf_helpers can also be removed. Please let me know. 
#define MAX_TAIL_CALL_CNT 33

Reply:
I agree